cmd/webhook/main.go

```diff
@@ -22,6 +15,14 @@ import (
 	"k8s.io/client-go/rest"
 	kubevirtv1 "kubevirt.io/api/core/v1"
 
+	ctlcni "github.com/harvester/harvester/pkg/generated/controllers/k8s.cni.cncf.io"
+	ctlcniv1 "github.com/harvester/harvester/pkg/generated/controllers/k8s.cni.cncf.io/v1"
+	ctlkubevirt "github.com/harvester/harvester/pkg/generated/controllers/kubevirt.io"
+	ctlkubevirtv1 "github.com/harvester/harvester/pkg/generated/controllers/kubevirt.io/v1"
+	"github.com/harvester/harvester/pkg/indexeres"
+	"github.com/harvester/webhook/pkg/config"
+	"github.com/harvester/webhook/pkg/server"
+
```

Comment:
Is there any reason to restructure the import order? A save in my IDE auto-formats the imports order back to the original one. The convention of import order:
```
// build in packages

// external

// packages in this module
```

Reply:
Thanks for reminding, my IDE settings is not set up correctly.